.github/workflows/sanity-deploy.yml

```diff
@@ -33,7 +29,7 @@ jobs:
         run: pnpm install
 
       - name: Deploy Sanity
-        run: pnpm run deploy
+        run: pnpm --filter "studio" run deploy
```

Comment:
`pnpm -F studio deploy` would the shorthand work here as well?

Reply:
Det er mulig, jeg foretrekker vanligvis å være mer verbose ettersom det gjør koden mer lesbar.